google/cloud/managedidentities/v1/resource.proto

```diff
@@ -71,7 +71,7 @@ message Domain {
   map<string, string> labels = 2 [(google.api.field_behavior) = OPTIONAL];
 
   // Optional. The full names of the Google Compute Engine
-  // [networks](/compute/docs/networks-and-firewalls#networks) the domain
+  // [networks](https://cloud.google.com/compute/docs/networks-and-firewalls#networks) the domain
```

Comment:
@danoscarmike do you know if we would need to make this change in the Redis protos instead?

Reply:
Yep, we should do this upstream. This change will get blown away by the next sync.

Reply:
@frankyn just checking in - anyone cut a CL here yet?

Reply:
No one has cut a CL yet. Apologies, I missed these two last responses.

Reply:
Looks like it got resolved in: https://github.com/googleapis/googleapis/commit/8be3770465a6cf19bea97a84b3e82b113b0239b6